app/assets/javascripts/angular/controllers/mobile-dashboard.js

```diff
@@ -0,0 +1,200 @@
+angular.module('mobileApp', ['timer', 'truncate'])
+  .controller('mobileCtrl', ['$scope', '$http', '$interval', '$window', function($scope, $http, $interval, $window) {
+
+  // Initializations
+  $scope.nextRound = new moment();
+  $scope.news = [];
+
+  // Tab changing
+  $scope.focusedTab = 'news';
+
+  $scope.setTab = function(tab) {
+    $scope.focusedTab = tab;
+  }
+
+  // Some stuff shouldn't change. Eg: My Role, My Country.
+  $scope.myCountry = "Germany";
+  $scope.myRole = "Head of State";
```

Comment:
This stuff we should only have to query for once, so maybe it should be separate from the recurring API call?